WcaOnRails/app/views/results/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:title, t(".title")) %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <%= form_tag results_rankings_path, method: :get, remote: true, class: "results-select form-inline", id: "results-query-form" do %>
```

Comment:
At the moment yes, I'm using plain links for everything, although I kept this because:

1. I didn't know if we would want to keep the direct links or investigate other possible solutions.
2. The regions selector is not working at the moment, and I still don't know how to make it work :stuck_out_tongue: I could turn it into a dropdown menu with `link_to`s as the others if you think that's a good option.

Reply:
I think links are just fine in this case. For the regions selector you can either try turning options into links, or add a piece of JS that simply changes `window.location` depending on the selected option value. Either way, we don't use a form for that.

Reply:
I added a piece of JS to take care of that and removed everything related to the form :slightly_smiling_face: